packages/compiler-cli/src/ngtsc/annotations/src/component.ts

```diff
@@ -991,6 +1066,49 @@ export class ComponentDecoratorHandler implements
 
     this.cycleAnalyzer.recordSyntheticImport(origin, imported);
   }
+
+  /**
+   * Resolve the url of a resource relative to the file that contains the reference to it.
+   *
+   * Throws a FatalDiagnosticError when unable to resolve the file.
+   */
+  private _resolveResourceOrThrow(
+      file: string, basePath: string, nodeForError: ts.Node,
+      resourceType: ResourceTypeForDiagnostics): string {
+    try {
+      return this.resourceLoader.resolve(file, basePath);
+    } catch (e) {
```

Comment:
If `e` is unused here, you should be able to omit it
```suggestion
    } catch {
```

Reply:
That was more intended as a comment for the reviewers to explain why I did a (small) refactor here. For future readers of this code, the type signature explains what `isStringArray` does.

Reply:
I think you responded to my other comment on this thread